pkg/distributor/distributor.go

```diff
@@ -1501,20 +1509,20 @@ func toLabelValuesCardinalityRequest(labelNames []model.LabelName, matchers []*l
 }
 
 type labelValuesCardinalityConcurrentMap struct {
-	cardinalityMap map[string]map[string]uint64
-	lock           sync.Mutex
+	// cardinalityMapByZone stores a result for each zone.
```

Comment:
nit: I expected that zone is the first key but it seems like it's the last

```suggestion
	// cardinalityMapByZone stores a result for each zone. map[label_name]map[label_value]map[zone]
```

Reply:
Thanks, I had originally written it that way, but since we need to compare all zones for each label value, this structure made it easier to get them all in one place for `approximateFromZones`